manifests/init.pp

```diff
@@ -21,7 +128,7 @@
   $config_dir_group    = 'root',
   $config_dir_mode     = '0755',
   $config_ensure       = 'file',
-  $config_path         = '/etc/eyaml/config.yaml',
```

Comment:
I didn't meant to change the location just wanted the config file to be related to the config dir but i missed the it was using the class settings. Can we make it a local variable instead?